sdk/core/tests.yml

```diff
@@ -0,0 +1,7 @@
+trigger: none
+
+variables:
+  ProjectFile: sdk/core/Azure.Core/Azure.Core.sln
```

Comment:
Should we run all tests for the core projects?

Reply:
Core has one live test for `HttpPipeline` ... but since it's under samples I can see good reasons for not including it here... 

https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core/tests/samples/Sample1_HelloWorld.cs#L13

Reply:
@pakrym What do you think is a good set of test to run nightly for Azure.Core? We will be running all the other tests as part of their test pipeline so I don't think we need to duplicate them with this pipeline so perhaps all we do is run the core tests only.

Reply:
I don't think there is any need to duplicate the test runs. Core tests only seem fine.